templates/vanilla/packages/client/src/mud/getNetworkConfig.ts

```diff
@@ -0,0 +1,54 @@
+import { SetupContractConfig, getBurnerWallet } from "@latticexyz/std-client";
+import { foundry } from "@wagmi/chains";
+import latticeTestnet from "./supportedChains/latticeTestnet";
+import latestLatticeTestnetDeploy from "contracts/deploys/4242/latest.json";
+import latestLocalhostDeploy from "contracts/deploys/31337/latest.json";
+
+type NetworkConfig = SetupContractConfig & {
+  privateKey: string;
+  faucetServiceUrl?: string;
+};
+
+export async function getNetworkConfig(): Promise<NetworkConfig> {
+  const params = new URLSearchParams(window.location.search);
+
+  const supportedChains = [foundry, latticeTestnet];
```

Comment:
```suggestion
  const supportedChains: MudChain[] = [foundry, latticeTestnet];
```
to prevent typescript from complaining about `modeUrl` / `faucetUrl` missing on `foundry`